### ISSUE TITLE
Do you have a patreon page or an Amazon wishlist?

### ISSUE THREAD

**Issue by maglub** [KevinSheedy/chrome-tab-rotate#28]
Hi,

I am very happy with this plugin, after you merged the local config file pull request.

Is there any way I can show my appreciation? May I send you a small gift of sorts, or drop a tenner in a jar at your favorite pub?

**Comment by KevinSheedy** [comment]
Hi @maglub 
I'm glad you like the app and thanks for helping. I haven't used Patreon before but on your suggestion I just setup a page. If you have any suggestions for edits to the page that would be much appreciated:
https://www.patreon.com/kevdev

A positive review on the Chrome web store would also be very much appreciated:
https://chrome.google.com/webstore/detail/tab-rotate/pjgjpabbgnnoohijnillgbckikfkbjed

Finally, I just released a substantial refactor of the app. I ported the code to the latest version of javascript (es6) and updated all the 3rd party libraries. If you felt like testing out the latest version, 0.2.0, that would be really helpful.
